### DISCUSSION TITLE
help !!! when I use eslint to format my code,I got the error,how can I solve it ?

### DISCUSSION THREAD

**Discussion by ziyouzhea** [eslint/eslint#18699] [Q&A]
There was a problem loading formatter: E:\carCloudSystem\carbond-content-admin-web\node_modules\eslint\lib\cli-engine\formatters\stylish
Error: require() of ES Module E:\carCloudSystem\carbond-content-admin-web\node_modules\strip-ansi\index.js from E:\carCloudSystem\carbond-content-admin-web\node_modules\eslint\lib\cli-engine\formatters\stylish.js not supported.
Instead change the require of index.js in E:\carCloudSystem\carbond-content-admin-web\node_modules\eslint\lib\cli-engine\formatters\stylish.js to a dynamic import() which is available in all CommonJS modules.

**Comment by aladdin-add** [comment]
Based on the error message, it looks like you have installed a version of strip-ansi that is not compatible with eslint ( strip-ansi v7 is esm-only, and cannot be required by eslint).

In most cases this is a bug in the package manager, you can try deleting the lock file and try reinstalling it.
